template/config.tpl/tyk/confs/api_candig.json.tpl

```diff
@@ -0,0 +1,157 @@
+{
+    "api_id": "21",
+    "jwt_issued_at_validation_skew": 0,
+    "jwt_expires_at_validation_skew": 0,
+    "upstream_certificates": {},
+    "use_keyless": false,
+    "enable_coprocess_auth": false,
+    "base_identity_provided_by": "",
+    
+    "proxy": {
+	"target_url": "${LOCAL_CANDIG_SERVER}",
+	"strip_listen_path": true,
+        "disable_strip_slash": false,
+	"listen_path": "/${TYK_LISTEN_PATH}",
+        "transport": {
+            "ssl_insecure_skip_verify": false,
+            "ssl_ciphers": [],
+            "ssl_min_version": 0,
+            "proxy_url": ""
+        },
+        "target_list": [],
+        "preserve_host_header": false
+    },
+
+    "version_data": {
+	"not_versioned": true,
+	"versions": {
+	    "Default": {
+		"name": "Default",
+		"use_extended_paths": true
+	    }
+	},
+        "extended_paths": {
+            "ignored": [
+                {
+                    "path": "${KC_LOGIN_REDIRECT_PATH}",
+                    "method_actions": {
+                        "GET": {
+                            "action": "no_action",
+                            "code": 200,
+                            "headers": {}
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "custom_middleware": {
+	"pre": [
+            {
+	        "name": "authMiddleware",
+	        "path": "/opt/tyk-gateway/middleware/authMiddleware.js",
+	        "require_session": false
+	    }
+        ],
+	"post": [
+            {
+	        "name": "oidcDistributedClaimsConduitMiddleware",
+	        "path": "/opt/tyk-gateway/middleware/oidcDistributedClaimsConduitMiddleware.js",
+	        "require_session": false
+	    }
+        ],
+	"id_extractor": {
+	    "extract_with": "",
+	    "extract_from": "",
+	    "extractor_config": {}
+	},
+	"driver": "",
+	"auth_check": {
+	    "path": "",
+	    "require_session": false,
+	    "name": ""
+	},
+	"post_key_auth": [],
+	"response": []
+    },
+    
+    "config_data": {
+	"SESSION_ENDPOINTS": [
+	    "/",
+	    "/gene_search",
+	    "/patients_overview",
+	    "/sample_analysis",
+	    "/custom_visualization",
+	    "/api_info",
+	    "/serverinfo"
+	],
+	"TYK_SERVER": "${CANDIG_PUBLIC_URL}${CD_PUB_PORT}"
+    },
+    "openid_options": {
+	"segregate_by_client": false,
+	"providers": [
+            {
+                "issuer": "${KC_PUBLIC_URL}${KC_PUB_PORT}/auth/realms/${KC_REALM}",
+                "client_ids": {
+                    "Y3FfY2FuZGln": "policy21"
```

Comment:
That is the base64 of something? Is it always the same and really need to be hardcoded?

Reply:
So I am afraid that I let this slip in. It is your `cq_candig` :)

Reply:
I should perhaps make it dynamic.